src/pages/customize.js

```diff
@@ -12,6 +12,14 @@
         const theme = e.target.value;
         ipcRenderer.send('pref-change', theme);
     });
-
     themePicker.value = currentTheme;
+    
+    const currentZoom = prefs.zoom || 100;
+    const zoomSetting = document.getElementById("zoom");
```

Comment:
id rather use onblur here or at least debounce the keyup so that its not zooming for every number you type

Reply:
I started with blur but I switched to keyup. The "reasonable zoom settings" make it so that it zooms at the most likely zoom (not at 15 but yes at 150). Switched back to blur and it still works OK but you will need to "tab out" to see the results (I liked being able to "preview" the zoom before switching out of the box).